libevmasm/ExpressionClasses.cpp

```diff
@@ -86,7 +86,7 @@ ExpressionClasses::Id ExpressionClasses::find(
 		exp.id = id;
 	else
 	{
-		exp.id = m_representatives.size();
+		exp.id = static_cast<unsigned>(m_representatives.size());
```

Comment:
This code seems to be using `Id` as a type alias for `unsigned` so let's use it consistently:
```suggestion
		exp.id = static_cast<Id>(m_representatives.size());
```

Reply:
I know, this is changed back in of the commits where I tried to figure if this is what causes the gcc 7.5.0 crash.